libs/wingsdk/src/target-sim/tokens.ts

```diff
@@ -61,10 +60,6 @@ export class SimTokens extends Tokens {
    * Binds the given token to the host.
    */
   public bindValue(host: IInflightHost, value: any) {
-    if (!(host instanceof Function)) {
```

Comment:
I wonder if it'll be better to remove this restriction only on the sim right now (because it was tested only for the sim) and to open an issue for the rest of the targets

Reply:
maybe we can also narrow the restriction to NOT function and NOT a container? (instead of removing it completely?)